tools/delete_imaging_upload.pl

```diff
@@ -129,6 +134,11 @@ =head1 DESCRIPTION
    an error message and abort, leaving the database and file system untouched, if:
        a) A MINC file should have a corresponding defaced file but does not.
        b) A MINC file that has been defaced has an associated file that is not a defaced file.
+
+4. Delete a specific file that exists in table C<files> or table C<parameter_file> (in which case it will be associated to parameter
+   C<check_pic_filename>, C<check_nii_filename>, C<check_bval_filename> or C<check_bvec_filename>). Note that once a file is
+   deleted, all the files that have been derived/built using this deleted file will also be deleted. Use option C<-file fileBaseName>, 
+   where C<fileBaseName> is the basename of the file to delete, along with option C<-uploadID> to use the script this way.
```

Comment:
what happens if the file has a link to the `mri_violations_log` when a caveat is set to `warning`?

Reply:
hehe, I think I answered my question by reading the code. Looks like this is handled. Just let me know if I got it right.

Reply:
Yeah, files in `mri_violations_log` are deleted too.